### PR TITLE
Make error page responsive

### LINKED ISSUE #16: Update Error page to fit on mobile
As a user, when I do something I shouldn’t on my mobile, I should see an error page that fits the screen.

# Replication steps 

1. Type: http://localhost:3000/xyz  and press ‘enter’.

Or

1. Open the menu.

2. Click in the search box.

3. Enter the value: 100000000000000000000000000.

4. Press ‘enter’.

The desktop version of the error page is shown which is not mobile friendly.

# Prerequisites

You will need to know basic css for this task.

# Repository link

https://github.com/cennznet/explorer/tree/master.  

This explorer repository contains several projects. 

The project you need to work on is in the ‘portal’ folder.

# Design

Please refer to the InVision prototype here:  [UNcover Bounty Tasks Error Page](https://projects.invisionapp.com/share/73TY7JHJHRY#/screens/391032476_Error)

You will need to request access permission to access inspect mode on the InVision prototype. 

Screenshot of intended design also attached. 

# Get Up & Running

Please check the README and CONTRIBUTING files:

https://github.com/cennznet/explorer/blob/master/portal/README.md   https://github.com/cennznet/explorer/blob/master/portal/CONTRIBUTING.md  

# Guidelines & Hints

The current system uses the ‘error or not-found’ template and English localised text from locals\en.json. These do not have to be changed.  

You will have to add some custom mobile  css to _mobile.scss.

# Test & Verify Your Changes 

## Mobile Devices

Ensure your changes work on:

The latest Android and iPhones phones.

Chrome, Safari and FireFox responsive views.

Ensure your changes work as expected and look exactly like the attached design.

Also ensure that the changes do not introduce bugs or break the desktop version.

## Expected Outcome

**Given** user uses search in dropdown menu 

**When** user inputs invalid search parameter 

**Then** user should see the error page that fits on a mobile screen

**Given** user uses the browser search bar

**When** user enters an invalid url

**Then** user should see the error page that fits on a mobile screen

**Given** user uses the system

**When** user triggers an error

**Then** user should see the error page that fits on a mobile screen

## If you have any questions, please don’t hesitate to contact our team on [Slack](https://join.slack.com/t/centralitydev/shared_invite/enQtNjk5NjE1MDI5NjgzLWJlOTM2NjMxNTMyZDY0OGM0MjY2YjQ1YjJkNTk2ZTIzNTFmYmM3OTIwZWE3NzNhMzE4Mjg4MmY1YzBiZDk0ZDE).

Current Screenshot
image

Designed Screen (what your design should look like) 
[UNcover Bounty Tasks Error Page](https://projects.invisionapp.com/share/73TY7JHJHRY#/screens/391032476_Error)

### PR DESCRIPTION
**Pull request by mul53**
Adds mobile styles to the error page to make it responsive on mobile devices.
Closes #16 

*Would like some feedback on the footer, the current background and color of the footer don't much.

### CODE REVIEW

**Comment by mul53** [comment]
i added a gradient to the footer, if it needs a tweak let me know. I did rebase from master when creating this branch. The branch is up to date.

File: portal/src/assets/style/_mobile.scss [hunk 1]
```diff
@@ -724,6 +724,25 @@ hr {
...
+      h1 {
+        font-size: 1.75rem;
+      }
+      p {
+        font-size: 1rem;
+        color: white;
+      }
+    }
+  }
 }
```
Comment (nickjohngray) [inline portal/src/assets/style/_mobile.scss]: can you please add this style for page not found 

`  .not-found-page{
    background-size: cover;
    background-position: center;

    .mobile-main-content {
      min-height: 300px;
    }

    .not-found {
      h1 {
        font-size: 1.75rem;
      }
      p {
        font-size: 1rem;
        color: white;
      }
    }
  }`

**Comment by mul53** [comment]
I have made the changes

**Comment by nickjohngray** [comment]
LGTM

**Pull request merged**